### ISSUE TITLE
@graphql-codegen/client-preset-swc-plugin panics when used with Next v13

### ISSUE THREAD

**Issue by kamilkisiela** [dotansimha/graphql-code-generator#9326]
### Which packages are impacted by your issue?

@graphql-codegen/client-preset

### Describe the bug

Next fails to build the app when `@graphql-codegen/client-preset-swc-plugin` is defined in `experimental.swcPlugins`.

This is the output:
```
info  - Creating an optimized production build ...[1]    47505 bus error  yarn build
```

### Your Example Website or App

https://github.com/dotansimha/graphql-code-generator/tree/master/examples/react/nextjs-swr

### Steps to Reproduce the Bug or Issue

1. yarn build

### Expected behavior

1. yarn build
2. happy Kamil

### Screenshots or Videos

<img width="516" alt="Screenshot 2023-04-20 at 16 25 23" src="https://user-images.githubusercontent.com/8167190/233396810-bb8d426d-05c6-46fc-9619-2aca8384445c.png">

https://user-images.githubusercontent.com/8167190/233396912-ccc30e5e-0aba-4f3b-9782-b328582b561f.mov

### Platform

- OS: [e.g. macOS, Windows, Linux]
- NodeJS: v18.16.0 and v18.12.1
- `graphql` version: 16.6.0
- `@graphql-codegen/cli`: ^3.3.0
- `@graphql-codegen/client-preset`: ^3.0.0
- `@graphql-codegen/schema-ast`: 3.0.1
- `@graphql-codegen/client-preset-swc-plugin`: 0.1.3

### Codegen Config File

```js
// eslint-disable-next-line import/no-extraneous-dependencies
import { CodegenConfig } from '@graphql-codegen/cli';

const config: CodegenConfig = {
  schema: 'https://swapi-graphql.netlify.app/.netlify/functions/index',
  documents: ['components/**/*.tsx', 'pages/**/*.tsx'],
  generates: {
    './gql/': {
      preset: 'client',
    },
  },
  hooks: { afterAllFileWrite: ['prettier --write'] },
};

export default config;
```

**Comment by kaorun343** [comment]
it seems updating swc_core is needed.

https://github.com/dotansimha/graphql-code-generator/blob/c0f9e9f3381b31a9f7cba0c236f7f853ff8f74c1/packages/presets/swc-plugin/Cargo.toml#L30

**Comment by kamilkisiela** [comment]
Fixed by #9335 and #9349
